.github/labeler.yml

```diff
@@ -0,0 +1,38 @@
+lib:
+  - lib/**/*
+  - include/**/*
+
+core:
+  - db/**/*
+  - display/**/*
+  - general/**/*
+  - imagery/**/*
+  - misc/**/*
+  - ps/**/*
+  - raster/**/*
+  - raster3d/**/*
+  - scripts/**/*
+  - temporal/**/*
+  - vector/**/*
+
+GUI:
+  - gui/**/*
+
+Windows:
+  - mswindows/**/*
+
+MacOS:
+  - macosx/**/*
+
+Linux:
+  - rpm/**/*
+  - singularity/**/*
+
+docker:
+  - docker/**/*
+
+docs:
```

Comment:
We have already a label called `manual`. Either use `manual` or renamed existing label to `docs`.

Reply:
We were prefixing some commits with `docs:` and `doc:` in the past. `docs` is often used in Git repos for the documentation directory. `docs` might be broader than `manual`, so perhaps more useful as it can be applied to other things than just `doc` dir.

Reply:
Shall we rename the current label https://github.com/OSGeo/grass/labels/manual to docs? (fine for me)